utils/vlog/vlog_to_pbtype.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+import yosys_exec
+import lxml.etree as ET
+import argparse, re
+import os, tempfile
+from yosys_json import YosysJson
+
+"""
+Convert a Verilog simulation model to a VPR `pb_type.xml`
+
+The following Verilog attributes are allowed on instances:
```

Comment:
I don't think we want a `PB_PATH` attribute. That should be specified on the command line?